.github/workflows/supress_warning.py

```
@@ -0,0 +1,53 @@
+# Dictionary of fields that should not appear in translation check warnings
+# per each language
+#
+# each maintainer kindly PR's STR_nnnn he has checked and found being reported
+# incorectly
+
+
+class DictWithoutKeyError(dict):
+# this is to eliminate KeyError that could occur when acessing
+# language not listed here
+    def __missing__(self, key):
+        return []
```

Do we need this? We should have keys for all languages, right? Letting the error in will remind us to add the new key here when we add a new language.

---

Okay, I was not really sure if you would like to prefer "hiding problems" or "getting problems into face" approach, metaphoricaly said. 
Do you think we need to try to catch possibly created exception so it gets printed in output or leave it without catching and just let script break in case of new language?

---

I'd rather have this blow-up, it's very unlikely for us to be looking at the logs of this CI already and this would fail silently.